### ISSUE TITLE
[FR] Navigate the search suggestions with the arrow keys and enter

### ISSUE THREAD

**Issue by DarkIntaqt** [DarkIntaqt/challenges#152]
On the start search bar. 
Could also be #136 #142

**Comment by BlossomiShymae** [comment]
Hiya! Is it this search bar here?

![image](https://user-images.githubusercontent.com/87099578/229322881-465d0f32-338c-4cdf-ad0d-f3df254a53df.png)

If I press TAB right now when the input bar is focused, it closes the search results dialog. Pressing TAB should probably focus on the search results dialog so the arrow keys can work :3

**Comment by DarkIntaqt** [comment]
Yeah that currently is the search bar :/ It currently adds a class of the search input is focused, on blur the class is removed again. This is probably not optimal
